components/binding-usage-controller/docs/usage-kind.md

```diff
@@ -0,0 +1,33 @@
+# UsageKind
+## Overview
+
+UsageKind allows dynamic, in the runtime, Binding Usage Controller configuration. It allows you to define what resources can be bound with ServiceBinding and how to bind them.
```

Comment:
`It allows you to define which resources can be bound with the ServiceBinding and how to bind them.`